extensions/src/main/java/org/pentaho/platform/plugin/action/ActionInvokerSystemListener.java

```diff
@@ -202,7 +220,12 @@ private void buildPayload( String urlEncodedJson, String enc ) {
     }
 
     IActionInvokeStatus issueRequest( ) throws Exception {
+      publishWorkItemStatus( WorkItemLifecyclePhase.RECEIVED, null );
```

Comment:
Why does the "Received" status first get logged here?  It seems to me if there are failure paths which occur before this, received should occur before those.  Not a blocker to merging, just a question.